salt/metalk8s/salt/master/configured.sls

```diff
@@ -13,10 +13,7 @@ Configure salt master:
     - template: jinja
     - defaults:
         salt_ip: "{{ salt_ip }}"
-        kubeconfig: "{{ pillar['metalk8s']['api_server']['kubeconfig'] }}"
-        {%- if pillar['metalk8s']['api_server'].get('context') %}
-        kubecontext: "{{ pillar['metalk8s']['api_server']['context'] }}"
-        {%- endif %}
+        kubeconfig: "/etc/kubernetes/admin.conf"
```

Comment:
Like already said on slack for me this change should land in `dev/2.0` to not have to edit `scripts/iso-manager.sh` in this PR

Reply:
I'll do a separate PR to handle that. Don't want this one to go in 2.0, and don't want to wait for something to land in 2.0 which is primarily 'cosmetic'.